cmd/osv-reporter/main.go

```diff
@@ -160,9 +165,14 @@ func run(args []string, stdout, stderr io.Writer) int {
 				}
 			}
 
+			failOnVuln := true // Default true
```

Comment:
nit: can we just simplify this to:

```golang
failOnVuln := !context.IsSet("fail-on-vuln") || context.Bool("fail-on-vuln")
```

?

Reply:
Yep